python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -183,6 +184,19 @@ cdef class RandomForest_impl():
         else:
             return CRITERION_END
 
+    def _get_max_feat_val(self):
+        if type(self.max_features) == int:
+            max_feature_val = 1/(self.max_features)
+        elif type(self.max_features) == float:
+            max_feature_val = self.max_features
+        elif self.max_features == 'sqrt':
+            max_feature_val = 1/np.sqrt(self.n_cols)
+        elif self.max_features == 'log2':
+            max_feature_val = 1/math.log2(self.n_cols)
+        else:
+            max_feature_val = 1.0
```

Comment:
Could you please expand the "sqrt" clause to also include the `"auto"` option, so as to be in sync with sklearn?
Also, maybe make explicit that the `else` case is capturing the `None` option?

Reply:
in the sklearn API the max_features value for when max_features = auto is:
```
max_features : int, float, string or None, optional (default=”auto”)
The number of features to consider when looking for the best split:

If int, then consider max_features features at each split.
If float, then max_features is a fraction and int(max_features * n_features) features are considered at each split.
If “auto”, then max_features=n_features.
If “sqrt”, then max_features=sqrt(n_features).
If “log2”, then max_features=log2(n_features).
If None, then max_features=n_features.
```
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html
which is the same as the None option and not sqrt.
I will edit the else option to explicitly show the option it accepts.

Reply:
My bad. My comment is only relevant for the classifier and not the regressor. Thanks for clarifying.